src/components/searchabletable.tsx

```diff
@@ -166,16 +170,11 @@ export class SearchableTable extends PureComponent<SearchableTableProps, Searcha
 						<Button icon onClick={() => this._onChangeFilter('')} >
 							<Icon name='delete' />
 						</Button>
-						<Popup
-							wide
-							trigger={<Button icon style={{ marginLeft: '1em' }}><Icon name="help" /></Button>}
-							header={'Advanced search'}
-							content={this.props.explanation}
-							position="bottom left"
-						/>
```

Comment:
By reverting to the the old code, you undo one of the changes from #77 to make the popup show properly on mobile devices. The design of the hover/button doesn't matter, but it needs `position="bottom left"` to display properly.